<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Publications | shashibp-lab</title>
    <meta name="author" content="Shashi B. Pandit" />
    <meta name="description" content="List of publications" />
    <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website" />
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet"
        integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css"
        integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous" />
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css"
        integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css"
        integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css"
        href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css"
        media="none" id="highlight_theme_light" />
    <link rel="shortcut icon" href="/assets/img/iscbl.favicon.svg" />
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://shashibp-lab.github.io/publications/">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css"
        media="none" id="highlight_theme_dark" />
    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script>
</head>

<body class="fixed-top-nav ">
    <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container"> <a class="navbar-brand" href="/"> <img src="/assets/img/grp_header.svg" alt=""
                    width="100"> </a> </div>
        <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse"
                data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
                <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span
                    class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button>
            <div class="collapse navbar-collapse text-right" id="navbarNav">
                <ul class="navbar-nav ml-auto flex-nowrap">
                    <li class="nav-item "> <a class="nav-link" href="/">Home</a> </li>
                    <li class="nav-item "> <a class="nav-link" href="/projects/">Research</a> </li>
                    <li class="nav-item active"> <a class="nav-link" href="/publications/">Publications<span
                                class="sr-only">(current)</span></a> </li>
                    <li class="nav-item "> <a class="nav-link" href="/members/">People</a> </li>
                    <li class="nav-item "> <a class="nav-link" href="/aboutus">About</a> </li>
                    <li class="nav-item "> <a class="nav-link" href="/position/">Position</a> </li>
                    <li class="nav-item "> <a class="nav-link" href="/teaching/">Teaching</a> </li>
                    <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown"
                            role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">More ..</a>
                        <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a
                                class="dropdown-item" href="/cv/">cv</a>
                            <div class="dropdown-divider"></div> <a class="dropdown-item"
                                href="/careerincompbio/">career in comp. bio.</a>
                        </div>
                    </li>
                    <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i
                                class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li>
                </ul>
            </div>
        </div>
    </nav>
    <header class="masthead">
        <div class="container-fluid h-100 border-bottom border-dark">
            <div class="row h-100 d-flex align-items-end">
                <div class="col-12 text-center"> <span class="align-bottom">
                        <h1 class="fw-bold bg-dark text-light">Integrative Systems Computational Biology Group</h1>
                    </span> </div>
            </div>
        </div>
    </header>
    <div class="container mt-5">
        <div class="post">
            <header class="post-header">
                <h1 class="post-title"><span style="text-transform: capitalize;">Publications</span></h1>
                <p class="post-description">List of publications</p>
            </header>
            <article>
                <div class="publications">
                    <h2 class="year">2025</h2>
                    <ol class="bibliography">
                        <li>
                            <div class="row">
                                <div class="col-sm-2 abbr"><abbr class="badge">bioRxiv</abbr></div>
                                <divRana" c5ass="col-sm-8">
                                    <div class="title">Deciphering Binding Site Conformational Variability of Substrate
                                        Promiscuous and Specialist Enzymes</div>
                                    <div class="author"> Deeksha Thakur, Paras Verma, and Shashi Bhushan Pandit</div>
                                    <div class="periodical"> <em>Journal of Chemical Information and Modeling</em> 2025
                                    </div>
                                    <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                        <a href="https://pubs.acs.org/doi/abs/10.1021/acs.jcim.5c01070"
                                            class="btn btn-sm z-depth-0" role="button" target="_blank"
                                            rel="noopener noreferrer">HTML</a>
                                    </div>
                                    <div class="abstract hidden">
                                        <p>Substrate promiscuity is the ability of enzymes to catalyze the same reaction
                                            with alternate substrates beyond their physiologically relevant cognate
                                            substrates. These promiscuous activities expand enzymes’ functional
                                            landscape, enabling evolution to catalyze novel biochemical reactions or the
                                            rational design of enzymes. The molecular basis of substrate promiscuity
                                            remains elusive, although recent studies have highlighted the significance
                                            of active site conformational variability in imparting promiscuity through
                                            alternate accessible conformational states. Accordingly, it has been
                                            hypothesized that promiscuous enzymes can accommodate alternate substrates
                                            through their pre-existing conformations. Specialist enzymes, known to
                                            catalyze reactions with a particular substrate, may exhibit a predominant
                                            conformation for their native substrates. To explore the role of active site
                                            conformational flexibility in substrate promiscuity, we compare the
                                            conformational binding competent substates in substrate promiscuous and
                                            specialist enzymes by analyzing long-time explicit water molecular dynamics.
                                            In addition to visually analyzing the variability of alternate
                                            conformations, we quantitatively assess this using the native functionality
                                            score, which measures the similarity of each conformation to the native
                                            substrate-binding residues. Our analysis of the two enzymes each from
                                            promiscuous and specialist groups shows multiple and restrictive
                                            substrate-binding competent conformers in both groups. Thus, conformational
                                            dynamics likely play a key role in enabling both enzyme groups to bind
                                            different substrates, albeit to varying extents.</p>
                                    </div>
                            </div>
                </div>
                </li>
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr"><abbr class="badge">bioRxiv</abbr></div>
                        <div id="Rana2025.07.15.603504" class="col-sm-8">
                            <div class="title">Genome assembly and annotation of the parasitoid jewel wasp Nasonia
                                oneida</div>
                            <div class="author"> Anjali Rana, Paras Verma, Shashi Bhushan Pandit, John H Werren, Xu
                                Wang, Rhitoban Raychoudhury</div>
                            <div class="periodical"> <em>Scientific Data</em> 2025
                            </div>
                            <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                <a href="https://pubmed.ncbi.nlm.nih.gov/40593889/" class="btn btn-sm z-depth-0"
                                    role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                            </div>
                            <div class="abstract hidden">
                                <p>The jewel wasp, Nasonia (Hymenoptera: Pteromalidae), is a well-established model
                                    system for evolutionary genetics and host-microbial interactions. Here, we present
                                    the genome of N. oneida, a species lacking prior genomic characterization, using 10×
                                    Genomics linked-read (400× coverage), Illumina short-read (120× coverage), and
                                    transcriptome data (30× coverage). The assembled genome size is 267 Mb, comprising
                                    4,675 scaffolds, with a scaffold N50 of 1 Mb and 98.40% Benchmarking Universal
                                    Single-Copy Orthologues (BUSCOs) completeness score. Annotation revealed 32.29%
                                    (86.46 Mb) of repetitive sequences and 14,221 protein-coding genes. Comparative
                                    genomics of N. oneida with 15 other hymenopteran species validated the presence of
                                    5,939 gene families shared among them, including 3643 single-copy and 2296 multicopy
                                    gene families. This study provides the first de novo assembly of N. oneida,
                                    providing a significant addition to the growing repertoire of molecular tools for
                                    comparative genomics and functional studies to understand the evolution of closely
                                    related species as well as the evolution of parasitic wasps.</p>
                            </div>
                        </div>
                    </div>
                </li>
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr"><abbr class="badge">bioRxiv</abbr></div>
                        <div id="Verma2024.06.07.597685" class="col-sm-8">
                            <div class="title">Exon Nomenclature And Classification of Transcripts (ENACT) provides a
                                systematic framework to annotate exon attributes</div>
                            <div class="author"> Paras Verma, Deeksha Thakur, Deepanshi Awasth, and Shashi Bhushan
                                Pandit</div>
                            <div class="periodical"> <em>Genome Research</em> 2025 </div>
                            <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                <a href="https://pubmed.ncbi.nlm.nih.gov/40335152/"
                                    class="btn btn-sm z-depth-0" role="button" target="_blank"
                                    rel="noopener noreferrer">HTML</a>
                            </div>
                            <div class="abstract hidden">
                                <p>Isoform diversity is known to enhance a gene's functional repertoire by producing
                                    protein variants with distinct functional implications. Despite numerous studies on
                                    transcriptome diversifying processes (alternative splicing/transcription),
                                    understanding their extent and correlated impact on proteome diversity remains
                                    limited owing to dearth of subsequent proteogenomic consequences. To coalesce the
                                    genomic information embedded in exons with isoform sequences, we present an
                                    innovative framework, "Exon Nomenclature And Classification of Transcripts" (ENACT).
                                    This centralizes exonic loci such that protein sequence information is integrated
                                    (onto the available/annotated or new transcripts) while enabling tracking and
                                    assessing splice-site variability through unique yielded descriptors. The resulting
                                    annotation from the ENACT framework enables exon features to be tractable,
                                    facilitating a systematic analysis of isoform diversity. Our findings and case
                                    studies unveil systemic exon inclusion roles in regulating diversity in coding
                                    region. Correspondingly, annotation of protein-coding genes and associated
                                    transcripts from C. elegans, D. melanogaster, D. rerio, M. musculus, and H. sapiens
                                    are publicly accessible in a dedicated resource.</p>
                            </div>
                        </div>
                    </div>
                </li>
                </ol>

                <h2 class="year">2024</h2>
                <ol class="bibliography">
                    <li>
                    <div class="row">
                        <div class="col-sm-2 abbr"><abbr class="badge">bioRxiv</abbr></div>
                        <div id="Verma2024.03.08.584087" class="col-sm-8">
                            <div class="title">Uncovering the translatome impact of transcriptome induced
                                diversity in eukaryotes: framework and innovative insights</div>
                            <div class="author"> Paras Verma, Deeksha Thakur, and Shashi Bhushan Pandit</div>
                            <div class="periodical"> <em>bioRxiv preprint</em> 2024 </div>
                            <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                <a href="https://www.biorxiv.org/content/early/2024/03/13/2024.03.08.584087"
                                    class="btn btn-sm z-depth-0" role="button" target="_blank"
                                    rel="noopener noreferrer">HTML</a>
                            </div>
                            <div class="abstract hidden">
                                <p>Isoform diversity is known to enhance a gene’s functional repertoire.
                                    Insights into the extent of such sequence variability generated through
                                    alternative splicing (AS), may unveil the layers of gene
                                    function/regulation. Despite studies on transcriptome diversifying
                                    processes, the impact of AS or related processes on sequence diversity still
                                    needs to be explored. Current study presents an innovative framework that
                                    centralizes exonic loci while integrating protein sequence per entity with
                                    attention to splice site variability assessment. The resulting framework
                                    enables exon (features) to be tractable, facilitating a systematic, detailed
                                    analysis of isoform diversity. We analyzed isoform diversity in five
                                    representative organisms and detailed the role of AS and related processes
                                    influencing exon inclusion in imparting sequence variation for human genome.
                                    Through analyses of exonic variations in two maximally diverged isoforms of
                                    human genes, we unraveled intricate splicing patterns prevalent in coding
                                    and non-coding regions. We observed that alternative splice sites, sequence
                                    changes, and skipping of exons are prevalent in coding exons, while the
                                    alternate first exon events are predominant in non-coding exons. Our
                                    findings offer a comprehensive understanding of isoform diversity as a
                                    function of exonic entity framework, providing valuable insights into the
                                    orchestration of exonic events in shaping the proteogenomic
                                    landscape.Competing Interest StatementThe authors have declared no competing
                                    interest.</p>
                            </div>
                        </div>
                    </div>
                </li>
                <li>
                    <div class="row">
                        <div class="col-sm-2 abbr"><abbr class="badge">Bioinfo. Adv.</abbr></div>
                        <div id="Verma2024" class="col-sm-8">
                            <div class="title">Exon nomenclature and classification of transcripts database
                                (ENACTdb): a resource for analyzing alternative splicing mediated proteome
                                diversity</div>
                            <div class="author"> Paras Verma, Deeksha Thakur, and Shashi B Pandit</div>
                            <div class="periodical"> <em>Bioinformatics Advances</em> Oct 2024 </div>
                            <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                <a href="https://doi.org/10.1093/bioadv/vbae157" class="btn btn-sm z-depth-0"
                                    role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                            </div>
                            <div class="abstract hidden">
                                <p>Gene transcripts are distinguished by the composition of their exons, and
                                    this different exon composition may contribute to advancing proteome
                                    complexity. Despite the availability of alternative splicing information
                                    documented in various databases, a ready association of exonic variations to
                                    the protein sequence remains a mammoth task.To associate exonic variation(s)
                                    with the protein systematically, we designed the Exon Nomenclature and
                                    Classification of Transcripts (ENACT) framework for uniquely annotating
                                    exons that tracks their loci in gene architecture context with encapsulating
                                    variations in splice site(s) and amino acid coding status. After ENACT
                                    annotation, predicted protein features (secondary structure/disorder/Pfam
                                    domains) are mapped to exon attributes. Thus, ENACTdb provides trackable
                                    exonic variation(s) association to isoform(s) and protein features, enabling
                                    the assessment of functional variation due to changes in exon composition.
                                    Such analyses can be readily performed through multiple views supported by
                                    the server. The exon-centric visualizations of ENACT annotated isoforms
                                    could provide insights on the functional repertoire of genes due to
                                    alternative splicing and its related processes and can serve as an important
                                    resource for the research community.The database is publicly available at
                                    https://www.iscbglab.in/enactdb/. It contains protein-coding genes and
                                    isoforms for Caenorhabditis elegans, Drosophila melanogaster, Danio rerio,
                                    Mus musculus, and Homo sapiens.</p>
                            </div>
                        </div>
                    </div>
                </li>
                </ol>


                <h2 class="year">2022</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">J. Struct. Biol.</abbr></div>
                            <div id="Thakur2022" class="col-sm-8">
                                <div class="title">Unusual commonality in active site structural features of
                                    substrate promiscuous and specialist enzymes</div>
                                <div class="author"> D. Thakur, and S.B. Pandit</div>
                                <div class="periodical"> <em>Journal of Structural Biology</em> Oct 2022 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1016/j.jsb.2022.107835" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Enzyme promiscuity is the ability of (some) enzymes to perform alternate
                                        reactions or catalyze non-cognate substrate(s). The latter is referred to as
                                        substrate promiscuity, widely studied for its biotechnological applications
                                        and understanding enzyme evolution. Insights into the structural basis of
                                        substrate promiscuity would greatly benefit the design and engineering of
                                        enzymes. Previous studies on some enzymes have suggested that flexibility,
                                        hydrophobicity, and active site protonation state could play an important
                                        role in enzyme promiscuity. However, it is not known yet whether substrate
                                        promiscuous enzymes have distinctive structural characteristics compared to
                                        specialist enzymes, which are specific for a substrate. In pursuit to
                                        address this, we have systematically compared substrate/catalytic binding
                                        site structural features of substrate promiscuous with those of specialist
                                        enzymes. For this, we have carefully constructed dataset of substrate
                                        promiscuous and specialist enzymes. On careful analysis, surprisingly, we
                                        found that substrate promiscuous and specialist enzymes are similar in
                                        various binding/catalytic site structural features such as flexibility,
                                        surface area, hydrophobicity, depth, and secondary structures. Recent
                                        studies have also alluded that promiscuity is widespread among enzymes.
                                        Based on these observations, we propose that substrate promiscuity could be
                                        defined as a continuum feature that varies from narrow (specialist) to broad
                                        range of substrate preferences. Moreover, diversity of conformational states
                                        of an enzyme accessible for ligand binding may possibly regulate its
                                        substrate preferences. © 2022 Elsevier Inc.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Front. Microbiol.</abbr></div>
                            <div id="Singh2022" class="col-sm-8">
                                <div class="title">Curcumin Inhibits Membrane-Damaging Pore-Forming Function of the
                                    β-Barrel Pore-Forming Toxin Vibrio cholerae Cytolysin</div>
                                <div class="author"> M. Singh, N. Rupesh, S.B. Pandit, and <span class="more-authors"
                                        title="click to view 1 more author"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'K. Chattopadhyay' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1
                                        more author</span> </div>
                                <div class="periodical"> <em>Frontiers in Microbiology</em> Oct 2022 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.3389/fmicb.2021.809782" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Vibrio cholerae cytolysin (VCC) is a β-barrel pore-forming toxin (β-PFT).
                                        Upon encountering the target cells, VCC forms heptameric β-barrel pores and
                                        permeabilizes the cell membranes. Structure-function mechanisms of VCC have
                                        been extensively studied in the past. However, the existence of any natural
                                        inhibitor for VCC has not been reported yet. In the present study, we show
                                        that curcumin can compromise the membrane-damaging activity of VCC. Curcumin
                                        is known to modulate a wide variety of biological processes and functions.
                                        However, the application of curcumin in the physiological scenario often
                                        gets limited due to its extremely poor solubility in the aqueous
                                        environment. Interestingly, we find that VCC can associate with the
                                        insoluble fraction of curcumin in the aqueous medium and thus gets separated
                                        from the solution phase. This, in turn, reduces the availability of VCC to
                                        attack the target membranes and thus blocks the membrane-damaging action of
                                        the toxin. We also observe that the soluble aqueous extract of curcumin,
                                        generated by the heat treatment, compromises the pore-forming activity of
                                        VCC. Interestingly, in the presence of such soluble extract of curcumin, VCC
                                        binds to the target membranes and forms the oligomeric assembly. However,
                                        such oligomers appear to be non-functional, devoid of the pore-forming
                                        activity. The ability of curcumin to bind to VCC and neutralize its
                                        membrane-damaging activity suggests that curcumin has the potential to act
                                        as an inhibitor of this potent bacterial β-PFT. Copyright © 2022 Singh,
                                        Rupesh, Pandit and Chattopadhyay.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2021</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">ACS Bio. Sci. Eng.</abbr></div>
                            <div id="Verma2021" class="col-sm-8">
                                <div class="title">Optimal Protein Sequence Design Mitigates Mechanical Failure in
                                    Silk β-Sheet Nanocrystals</div>
                                <div class="author"> P. Verma, B. Panda, K.P. Singh, and <span class="more-authors"
                                        title="click to view 1 more author"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'S.B. Pandit' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1
                                        more author</span> </div>
                                <div class="periodical"> <em>ACS Biomaterials Science and Engineering</em> Oct 2021
                                </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1021/acsbiomaterials.1c00447"
                                        class="btn btn-sm z-depth-0" role="button" target="_blank"
                                        rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>The excellent mechanical strength and toughness of spider silk are well
                                        characterized experimentally and understood atomistically using
                                        computational simulations. However, little attention has been focused on
                                        understanding whether the amino acid sequence of β-sheet nanocrystals, which
                                        is the key to rendering strength to silk fiber, is optimally chosen to
                                        mitigate molecular-scale failure mechanisms. To investigate this, we modeled
                                        β-sheet nanocrystals of various representative small/polar/hydrophobic amino
                                        acid repeats for determining the sequence motif having superior
                                        nanomechanical tensile strength and toughness. The constant velocity pulling
                                        of the central β-strand in the nanocrystal, using steered molecular
                                        dynamics, showed that homopolymers of small amino acid
                                        (alanine/alanine-glycine) sequence motifs, occurring in natural silk
                                        fibroin, have better nanomechanical properties than other modeled
                                        structures. Further, we analyzed the hydrogen bond (HB) and β-strand pull
                                        dynamics of modeled nanocrystals to understand the variation in their
                                        rupture mechanisms and explore sequence-dependent mitigating factors
                                        contributing to their superior mechanical properties. Surprisingly, the
                                        enhanced side-chain interactions in homopoly-polar/hydrophobic amino acid
                                        models are unable to augment backbone HB cooperativity to increase
                                        mechanical strength. Our analyses suggest that nanocrystals of pristine silk
                                        sequences most likely achieve superior mechanical strength by optimizing
                                        side-chain interaction, packing, and main-chain HB interactions. Thus, this
                                        study suggests that the nanocrystal β-sheet sequence plays a crucial role in
                                        determining the nanomechanical properties of silk, and the evolutionary
                                        process has optimized it in natural silk. This study provides insight into
                                        the molecular design principle of silk with implications in the genetically
                                        modified artificial synthesis of silk-like biomaterials. © 2021 American
                                        Chemical Society.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Mol. Microbiol.</abbr></div>
                            <div id="Mondal2021508" class="col-sm-8">
                                <div class="title">Tyrosine in the hinge region of the pore-forming motif regulates
                                    oligomeric β-barrel pore formation by Vibrio cholerae cytolysin</div>
                                <div class="author"> A.K. Mondal, P. Verma, N. Sengupta, and <span class="more-authors"
                                        title="click to view 3 more authors"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'S. Dutta, S. Bhushan Pandit, K. Chattopadhyay' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3
                                        more authors</span> </div>
                                <div class="periodical"> <em>Molecular Microbiology</em> Oct 2021 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1111/mmi.14631" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>β-barrel pore-forming toxins perforate cell membranes by forming oligomeric
                                        β-barrel pores. The most crucial step is the membrane-insertion of the
                                        pore-forming motifs that create the transmembrane β-barrel scaffold.
                                        Molecular mechanism that regulates structural reorganization of these
                                        pore-forming motifs during β-barrel pore-formation still remains elusive.
                                        Using Vibrio cholerae cytolysin as an archetypical example of the β-barrel
                                        pore-forming toxin, we show that a key tyrosine residue (Y321) in the hinge
                                        region of the pore-forming motif plays crucial role in this process.
                                        Mutation of Y321 abrogates oligomerization of the membrane-bound toxin
                                        protomers, and blocks subsequent steps of pore-formation. Our study suggests
                                        that the presence of Y321 in the hinge region of the pore-forming motif is
                                        crucial for the toxin molecule to sense membrane-binding, and to trigger
                                        essential structural rearrangements required for the subsequent
                                        oligomerization and pore-formation process. Such a regulatory mechanism of
                                        pore-formation by V. cholerae cytolysin has not been documented earlier in
                                        the structurally related β-barrel pore-forming toxins. © 2020 John Wiley
                                        &amp; Sons Ltd</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2020</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">BMB Educ.</abbr></div>
                            <div id="Bachhawat2020227" class="col-sm-8">
                                <div class="title">An inquiry-based approach in large undergraduate labs: Learning,
                                    by htmlng it the “wrong” way</div>
                                <div class="author"> A.K. Bachhawat, S.B. Pandit, I. Banerjee, and <span
                                        class="more-authors" title="click to view 4 more authors"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'S. Anand, R. Sarkar, A. Mrigwani, S.K. Mishra' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4
                                        more authors</span> </div>
                                <div class="periodical"> <em>Biochemistry and Molecular Biology Education</em> Oct
                                    2020 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1002/bmb.21331" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Undergraduate laboratory courses, owing to their larger sizes and shorter
                                        time slots, are often conducted in highly structured modes. However, this
                                        approach is known to interfere with students’ engagement in the experiments.
                                        To enhance students’ engagement, we propose an alternative mode of running
                                        laboratory courses by creating some “disorder” in a previously adopted
                                        structure. After performing an experiment in the right way, the students
                                        were asked to repeat the experiment but with a variation at certain steps
                                        leading to the experiment being done the “wrong” way. Although this approach
                                        led to fewer experiments being conducted in a semester, it significantly
                                        enhanced the students’ involvement. This was also reflected in the students’
                                        feedback. The majority of students preferred repeating an experiment with a
                                        variant protocol than performing a new experiment. Although we have tested
                                        this inquiry-based approach only for an undergraduate laboratory course in
                                        molecular biology, we believe such an approach could also be extended to
                                        undergraduate laboratory courses of other subjects. © 2020 International
                                        Union of Biochemistry and Molecular Biology</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2019</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">PLoS ONE</abbr></div>
                            <div id="Verma2019" class="col-sm-8">
                                <div class="title">Unraveling the structural landscape of intra-chain domain
                                    interfaces: Implication in the evolution of domain-domain interactions</div>
                                <div class="author"> R. Verma, and S.B. Pandit</div>
                                <div class="periodical"> <em>PLoS ONE</em> Oct 2019 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1371/journal.pone.0220336" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Intra-chain domain interactions are known to play a significant role in the
                                        function and stability of multidomain proteins. These interactions are
                                        mediated through a physical interaction at domain-domain interfaces (DDIs).
                                        With a motivation to understand evolution of interfaces, we have
                                        investigated similarities among DDIs. Even though interfaces of
                                        protein-protein interactions (PPIs) have been previously studied by
                                        structurally aligning interfaces, similar analyses have not yet been
                                        performed on DDIs of either multidomain proteins or PPIs. For studying the
                                        structural landscape of DDIs, we have used iAlign to structurally align
                                        intra-chain domain interfaces of domains. The interface alignment of
                                        spatially constrained domains (due to inter-domain linkers) showed that  88%
                                        of these could identify a structural matching interface having similar
                                        C-alpha geometry and contact pattern despite that aligned domain pairs are
                                        not structurally related. Moreover, the mean interface similarity score
                                        (IS-score) is 0.307, which is higher compared to the average random IS-score
                                        (0.207) suggesting domain interfaces are not random. The structural space of
                                        DDIs is highly connected as  84% of all possible directed edges among
                                        interfaces are found to have at most path length of 8 when 0.26 is IS-score
                                        threshold. At this threshold,  83% of interfaces form the largest strongly
                                        connected component. Thus, suggesting that structural space of intra-chain
                                        domain interfaces is degenerate and highly connected, as has been found in
                                        PPI interfaces. Interestingly, searching for structural neighbors of
                                        inter-chain interfaces among intra-chain interfaces showed that  86% could
                                        find a statistically significant match to intra-chain interface with a mean
                                        IS-score of 0.311. This implies that domain interfaces are degenerate
                                        whether formed within a protein or between proteins. The interface
                                        degeneracy is most likely due to limited possible ways of packing secondary
                                        structures. In principle, interface similarities can be exploited to
                                        accurately model domain interfaces in structure prediction of multidomain
                                        proteins. © 2019 Verma, Pandit. This is an open access article distributed
                                        under the terms of the Creative Commons Attribution License, which permits
                                        unrestricted use, distribution, and reproduction in any medium, provided the
                                        original author and source are credited.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2017</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">BMC Bioinform.</abbr></div>
                            <div id="Choudhary2017" class="col-sm-8">
                                <div class="title">CSmetaPred: A consensus method for prediction of catalytic
                                    residues</div>
                                <div class="author"> P. Choudhary, S. Kumar, A.K. Bachhawat, and <span
                                        class="more-authors" title="click to view 1 more author"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'S.B. Pandit' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1
                                        more author</span> </div>
                                <div class="periodical"> <em>BMC Bioinformatics</em> Oct 2017 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1186/s12859-017-1987-z" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Background: Knowledge of catalytic residues can play an essential role in
                                        elucidating mechanistic details of an enzyme. However, experimental
                                        identification of catalytic residues is a tedious and time-consuming task,
                                        which can be expedited by computational predictions. Despite significant
                                        development in active-site prediction methods, one of the remaining issues
                                        is ranked positions of putative catalytic residues among all ranked
                                        residues. In order to improve ranking of catalytic residues and their
                                        prediction accuracy, we have developed a meta-approach based method
                                        CSmetaPred. In this approach, residues are ranked based on the mean of
                                        normalized residue scores derived from four well-known catalytic residue
                                        predictors. The mean residue score of CSmetaPred is combined with predicted
                                        pocket information to improve prediction performance in meta-predictor,
                                        CSmetaPred_poc. Results: Both meta-predictors are evaluated on two
                                        comprehensive benchmark datasets and three legacy datasets using Receiver
                                        Operating Characteristic (ROC) and Precision Recall (PR) curves. The visual
                                        and quantitative analysis of ROC and PR curves shows that meta-predictors
                                        outperform their constituent methods and CSmetaPred_poc is the best of
                                        evaluated methods. For instance, on CSAMAC dataset CSmetaPred_poc
                                        (CSmetaPred) achieves highest Mean Average Specificity (MAS), a scalar
                                        measure for ROC curve, of 0.97 (0.96). Importantly, median predicted rank of
                                        catalytic residues is the lowest (best) for CSmetaPred_poc. Considering
                                        residues ranked ≤20 classified as true positive in binary classification,
                                        CSmetaPred_poc achieves prediction accuracy of 0.94 on CSAMAC dataset.
                                        Moreover, on the same dataset CSmetaPred_poc predicts all catalytic residues
                                        within top 20 ranks for  73% of enzymes. Furthermore, benchmarking of
                                        prediction on comparative modelled structures showed that models result in
                                        better prediction than only sequence based predictions. These analyses
                                        suggest that CSmetaPred_poc is able to rank putative catalytic residues at
                                        lower (better) ranked positions, which can facilitate and expedite their
                                        experimental characterization. Conclusions: The benchmarking studies showed
                                        that employing meta-approach in combining residue-level scores derived from
                                        well-known catalytic residue predictors can improve prediction accuracy as
                                        well as provide improved ranked positions of known catalytic residues.
                                        Hence, such predictions can assist experimentalist to prioritize residues
                                        for mutational studies in their efforts to characterize catalytic residues.
                                        Both meta-predictors are available as webserver at:
                                        http://14.139.227.206/csmetapred/. © 2017 The Author(s).</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Biochem. J.</abbr></div>
                            <div id="Kundu2017317" class="col-sm-8">
                                <div class="title">Disulphide bond restrains the C-terminal region of thermostable
                                    direct hemolysin during folding to promote oligomerization</div>
                                <div class="author"> N. Kundu, S. Tichkule, S.B. Pandit, and <span class="more-authors"
                                        title="click to view 1 more author"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'K. Chattopadhyay' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1
                                        more author</span> </div>
                                <div class="periodical"> <em>Biochemical Journal</em> Oct 2017 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1042/BCJ20160728" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Pore-forming toxins (PFTs) are typically produced as water-soluble monomers,
                                        which upon interacting with target cells assemble into transmembrane
                                        oligomeric pores. Vibrio parahaemolyticus thermostable direct hemolysin
                                        (TDH) is an atypical PFT that exists as a tetramer in solution, prior to
                                        membrane binding. The TDH structure highlights a core β-sandwich domain
                                        similar to those found in the eukaryotic actinoporin family of PFTs.
                                        However, the TDH structure harbors an extended C-terminal region (CTR) that
                                        is not documented in the actinoporins. This CTR remains tethered to the
                                        β-sandwich domain through an intra-molecular disulphide bond. Part of the
                                        CTR is positioned at the interprotomer interface in the TDH tetramer. Here
                                        we show that the truncation, as well as mutation, of the CTR compromise
                                        tetrameric assembly, and the membrane-damaging activity of TDH. Our study
                                        also reveals that intra-protomer disulphide bond formation during the
                                        folding/assembly process of TDH restrains the CTR to mediate its
                                        participation in the formation of inter-protomer contact, thus facilitating
                                        TDH oligomerization. However, once tetramerization is achieved, disruption
                                        of the disulphide bond does not affect oligomeric assembly. Our study
                                        provides critical insights regarding the regulation of the oligomerization
                                        mechanism of TDH, which has not been previously documented in the PFT
                                        family. © 2017 The Author(s); published by Portland Press Limited on behalf
                                        of the Biochemical Society.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2014</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Nucleic Acids Res.</abbr></div>
                            <div id="Carbonell2014" class="col-sm-8">
                                <div class="title">XTMS: Pathway design in an eXTended metabolic space</div>
                                <div class="author"> P. Carbonell, P. Parutto, J. Herisson, and <span
                                        class="more-authors" title="click to view 2 more authors"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'S.B. Pandit, J.-L. Faulon' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2
                                        more authors</span> </div>
                                <div class="periodical"> <em>Nucleic Acids Research</em> Oct 2014 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1093/nar/gku362" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>As metabolic engineering and synthetic biology progress toward reaching the
                                        goal of a more sustainable use of biological resources, the need of
                                        increasing the number of value-added chemicals that can be produced in
                                        industrial organisms becomes more imperative. Exploring, however, the vast
                                        possibility of pathways amenable to engineering through heterologous genes
                                        expression in a chassis organism is complex and unattainable manually. Here,
                                        we present XTMS, a web-based pathway analysis platform available at
                                        http://xtms.issb.genopole.fr, which provides full access to the set of
                                        pathways that can be imported into a chassis organism such as Escherichia
                                        coli through the application of an Extended Metabolic Space modeling
                                        framework. The XTMS approach consists on determining the set of biochemical
                                        transformations that can potentially be processed in vivo as modeled by
                                        molecular signatures, a specific coding system for derivation of reaction
                                        rules for metabolic reactions and enumeration of all the corresponding
                                        substrates and products. Most promising routes are described in terms of
                                        metabolite exchange, maximum allowable pathway yield, toxicity and enzyme
                                        efficiency. By answering such critical design points, XTMS not only paves
                                        the road toward the rationalization of metabolic engineering, but also opens
                                        new processing possibilities for non-natural metabolites and novel enzymatic
                                        transformations. © 2014 The Author(s).</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Mol. Biosyst.</abbr></div>
                            <div id="Tayal20141469" class="col-sm-8">
                                <div class="title">Evolutionarily conserved and conformationally constrained short
                                    peptides might serve as DNA recognition elements in intrinsically disordered
                                    regions</div>
                                <div class="author"> N. Tayal, P. Choudhary, S.B. Pandit, and <span class="more-authors"
                                        title="click to view 1 more author"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'K.S. Sandhu' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1
                                        more author</span> </div>
                                <div class="periodical"> <em>Molecular BioSystems</em> Oct 2014 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1039/c3mb70539k" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Despite recent advances, it is yet not clear how intrinsically disordered
                                        regions in proteins recognize their targets without any defined structures.
                                        Short linear motifs had been proposed to mediate molecular recognition by
                                        disordered regions; however, the underlying structural prerequisite remains
                                        elusive. Moreover, the role of short linear motifs in DNA recognition has
                                        not been studied. We report a repertoire of short evolutionarily Conserved
                                        Recognition Elements (CoREs) in long intrinsically disordered regions, which
                                        have very distinct amino-acid propensities from those of known motifs, and
                                        exhibit a strong tendency to retain their three-dimensional conformations
                                        compared to adjacent regions. The majority of CoREs directly interact with
                                        the DNA in the available 3D structures, which is further supported by
                                        literature evidence, analyses of ΔΔG values of DNA-binding energies and
                                        threading-based prediction of DNA binding potential. CoREs were enriched in
                                        cancer-associated missense mutations, further strengthening their functional
                                        nature. Significant enrichment of glycines in CoREs and the preference of
                                        glycyl φ-Ψ values within the left-handed bridge range in the l-disallowed
                                        region of the Ramachandran plot suggest that Gly-to-nonGly mutations within
                                        CoREs might alter the backbone conformation and consequently the function, a
                                        hypothesis that we reconciled using available mutation data. We conclude
                                        that CoREs might serve as bait for DNA recognition by long disordered
                                        regions and that certain mutations in these peptides can disrupt their DNA
                                        binding potential and consequently the protein function. We further
                                        hypothesize that the preferred conformations of CoREs and of glycyl residues
                                        therein might play an important role in DNA binding. The highly ordered
                                        nature of CoREs hints at a therapeutic strategy to inhibit malicious
                                        molecular interactions using small molecules mimicking CoRE conformations. ©
                                        2014 the Partner Organisations.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2012</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">BMC Syst. Biol.</abbr></div>
                            <div id="Carbonell2012" class="col-sm-8">
                                <div class="title">Enumerating metabolic pathways for the production of heterologous
                                    target chemicals in chassis organisms</div>
                                <div class="author"> P. Carbonell, D. Fichera, S.B. Pandit, and <span
                                        class="more-authors" title="click to view 1 more author"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'J.-L. Faulon' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1
                                        more author</span> </div>
                                <div class="periodical"> <em>BMC Systems Biology</em> Oct 2012 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1186/1752-0509-6-10" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Background: We consider the possibility of engineering metabolic pathways in
                                        a chassis organism in order to synthesize novel target compounds that are
                                        heterologous to the chassis. For this purpose, we model metabolic networks
                                        through hypergraphs where reactions are represented by hyperarcs. Each
                                        hyperarc represents an enzyme-catalyzed reaction that transforms set of
                                        substrates compounds into product compounds. We follow a retrosynthetic
                                        approach in order to search in the metabolic space (hypergraphs) for
                                        pathways (hyperpaths) linking the target compounds to a source set of
                                        compounds.Results: To select the best pathways to engineer, we have
                                        developed an objective function that computes the cost of inserting a
                                        heterologous pathway in a given chassis organism. In order to find
                                        minimum-cost pathways, we propose in this paper two methods based on steady
                                        state analysis and network topology that are to the best of our knowledge,
                                        the first to enumerate all possible heterologous pathways linking a target
                                        compounds to a source set of compounds. In the context of metabolic
                                        engineering, the source set is composed of all naturally produced chassis
                                        compounds (endogenuous chassis metabolites) and the target set can be any
                                        compound of the chemical space. We also provide an algorithm for identifying
                                        precursors which can be supplied to the growth media in order to increase
                                        the number of ways to synthesize specific target compounds.Conclusions: We
                                        find the topological approach to be faster by several orders of magnitude
                                        than the steady state approach. Yet both methods are generally scalable in
                                        time with the number of pathways in the metabolic network. Therefore this
                                        work provides a powerful tool for pathway enumeration with direct
                                        application to biosynthetic pathway design. © 2012 Carbonell et al.;
                                        licensee BioMed Central Ltd.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2011</h2>
                <ol class="bibliography"></ol>
                <h2 class="year">2010</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Proteins</abbr></div>
                            <div id="Pandit20102769" class="col-sm-8">
                                <div class="title">TASSER_low-zsc: An approach to improve structure prediction using
                                    low z-score-ranked templates</div>
                                <div class="author"> S.B. Pandit, and J. Skolnick</div>
                                <div class="periodical"> <em>Proteins: Structure, Function and Bioinformatics</em>
                                    Oct 2010 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1002/prot.22791" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>In a variety of threading methods, often poorly ranked (low z-score)
                                        templates have good alignments. Here, a new method, TASSER_low-zsc that
                                        identifies these low z-score-ranked templates to improve protein structure
                                        prediction accuracy, is described. The approach consists of clustering of
                                        threading templates by affinity propagation on the basis of structural
                                        similarity (thread_cluster) followed by TASSER modeling, with final models
                                        selected by using a TASSER_QA variant. To establish the generality of the
                                        approach, templates provided by two threading methods, SP 3 and SPARKS 2,
                                        are examined. The SP 3 and SPARKS 2 benchmark datasets consist of 351 and
                                        357 medium/hard proteins (those with moderate to poor quality templates
                                        and/or alignments) of length ≤250 residues, respectively. For SP 3 medium
                                        and hard targets, using thread_cluster, the TM-scores of the best template
                                        improve by  4 and 9% over the original set (without low z-score templates)
                                        respectively; after TASSER modeling/refinement and ranking, the best model
                                        improves by  7 and 9% over the best model generated with the original
                                        template set. Moreover, TASSER_low-zsc generates 22% (43%) more foldable
                                        medium (hard) targets. Similar improvements are observed with low-ranked
                                        templates from SPARKS 2. The template clustering approach could be applied
                                        to other modeling methods that utilize multiple templates to improve
                                        structure prediction. © 2010 Wiley-Liss, Inc.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Bioinformatics</abbr></div>
                            <div id="Pandit2010687" class="col-sm-8">
                                <div class="title">PSiFR: An integrated resource for prediction of protein structure
                                    and function</div>
                                <div class="author"> S.B. Pandit, M. Brylinski, H. Zhou, and <span class="more-authors"
                                        title="click to view 3 more authors"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'M. Gao, A.K. Arakaki, J. Skolnick' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3
                                        more authors</span> </div>
                                <div class="periodical"> <em>Bioinformatics</em> Oct 2010 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1093/bioinformatics/btq006" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>In the post-genomic era, the annotation of protein function facilitates the
                                        understanding of various biological processes. To extend the range of
                                        function annotation methods to the twilight zone of sequence identity, we
                                        have developed approaches that exploit both protein tertiary structure
                                        and/or protein sequence evolutionary relationships. To serve the scientific
                                        community, we have integrated the structure prediction tools, TASSER,
                                        TASSER-Lite and METATASSER, and the functional inference tools, FINDSITE, a
                                        structure-based algorithm for binding site prediction, Gene Ontology
                                        molecular function inference and ligand screening, EFICAz2, a sequence-based
                                        approach to enzyme function inference and DBD-hunter, an algorithm for
                                        predicting DNA-binding proteins and associated DNA-binding residues, into a
                                        unified web resource, Protein Structure and Function prediction Resource
                                        (PSiFR). Availability and implementation: PSiFR is freely available for use
                                        on the web at http://psifr.cssb.biology.gatech.edu/. Contact:
                                        skolnick@gatech.edu © The Author 2010. Published by Oxford University Press.
                                    </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">BOOK</abbr></div>
                            <div id="Bhushan Pandit2010219" class="col-sm-8">
                                <div class="title">[Book Chap.] Tasser-Based Protein Structure Prediction</div>
                                <div class="author"> Shashi Bhushan Pandit, Hongyi Zhou, and Jeffrey Skolnick</div>
                                <div class="periodical"> Oct 2010 </div>
                                <div class="links"> <a href="http://doi.org/10.1002/9780470882207.ch10"
                                        class="btn btn-sm z-depth-0" role="button" target="_blank"
                                        rel="noopener noreferrer">HTML</a> </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2009</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">BOOK</abbr></div>
                            <div id="Mohanty2009257" class="col-sm-8">
                                <div class="title">[Book Chap.] Dynamics of protein-protein interaction network in
                                    plasmodium falciparum</div>
                                <div class="author"> S. Mohanty, S.B. Pandit, and N. Srinivasan</div>
                                <div class="periodical"> Oct 2009 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.4018/978-1-60566-398-2.ch015"
                                        class="btn btn-sm z-depth-0" role="button" target="_blank"
                                        rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Integration of organism-wide protein interactome data with information on
                                        expression of genes, cellular localization of proteins and their functions
                                        has proved extremely useful in developing biologically intuitive interaction
                                        networks. This chapter highlights the dynamics in protein interaction
                                        network across different stages in the lifecycle of Plasmodium falciparum, a
                                        malarial parasite, and the implication of the network dynamics in different
                                        physiological processes. The main focus of the chapter is the integration of
                                        information on experimentally derived interactions of P.falciparum proteins
                                        with expression data and analysis of the implications of interactions in
                                        different cellular processes. Extensive analysis has been made to quantify
                                        the interaction dynamics across various stages, as well as correlating it
                                        with the dynamics of the cellular pathways involving the interacting
                                        proteins. The authors’ analysis demonstrates the power of strategic
                                        integration of genome-wide datasets in extracting information on dynamics of
                                        biological pathways and processes. © 2009, IGI Global.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Proteins</abbr></div>
                            <div id="Zhou2009123" class="col-sm-8">
                                <div class="title">Performance of the Pro-sp3-TASSER server in CASPB</div>
                                <div class="author"> H. Zhou, S.B. Pandit, and J. Skolnick</div>
                                <div class="periodical"> <em>Proteins: Structure, Function and Bioinformatics</em>
                                    Oct 2009 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1002/prot.22501" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>The performance of the protein structure prediction server pro-sp3-TASSER in
                                        CASP8 is described. Compared to CASP7, the major improvement in prediction
                                        is in the quality of input models to TASSER. These improvements are due to
                                        the PRO-SP 3 threading method, the improved quality of contact predictions
                                        provided by TASSER-2.0, multiple short TASSER simulations for building the
                                        full-length model, and the accuracy of model selection using the TASSER-QA
                                        quality assessment method. Finally, we analyze the overall performance and
                                        highlight some successful predictions of the pro-sp3-TASSER server. © 2009
                                        WILEY-LISS, INC.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2008</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">BMC Bioinform.</abbr></div>
                            <div id="Pandit2008" class="col-sm-8">
                                <div class="title">Fr-TM-align: A new protein structural alignment method based on
                                    fragment alignments and the TM-score</div>
                                <div class="author"> S.B. Pandit, and J. Skolnick</div>
                                <div class="periodical"> <em>BMC Bioinformatics</em> Oct 2008 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1186/1471-2105-9-531" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Background: Protein tertiary structure comparisons are employed in various
                                        fields of contemporary structural biology. Most structure comparison methods
                                        involve generation of an initial seed alignment, which is extended and/or
                                        refined to provide the best structural superposition between a pair of
                                        protein structures as assessed by a structure comparison metric. One such
                                        metric, the TM-score, was recently introduced to provide a combined
                                        structure quality measure of the coordinate root mean square deviation
                                        between a pair of structures and coverage. Using the TM-score, the TM-align
                                        structure alignment algorithm was developed that was often found to have
                                        better accuracy and coverage than the most commonly used structural
                                        alignment programs; however, there were a number of situations when this was
                                        not true. Results: To further improve structure alignment quality, the
                                        Fr-TM-align algorithm has been developed where aligned fragment pairs are
                                        used to generate the initial seed alignments that are then refined using
                                        dynamic programming to maximize the TM-score. For the assessment of the
                                        structural alignment quality from Fr-TM-align in comparison to other
                                        programs such as CE and TM-align, we examined various alignment quality
                                        assessment scores such as PSI and TM-score. The assessment showed that the
                                        structural alignment quality from Fr-TM-align is better in comparison to
                                        both CE and TM-align. On average, the structural alignments generated using
                                        Fr-TM-align have a higher TM-score (∼9%) and coverage (∼7%) in comparison to
                                        those generated by TM-align. Fr-TM-align uses an exhaustive procedure to
                                        generate initial seed alignments. Hence, the algorithm is computationally
                                        more expensive than TM-align. Conclusion: Fr-TM-align, a new algorithm that
                                        employs fragment alignment and assembly provides better structural
                                        alignments in comparison to TM-align. The source code and executables of
                                        Fr-TM-align are freely downloadable at:
                                        http://cssb.biology.gatech.edu/skolnick/files/ FrTMalign/. © 2008 Pandit and
                                        Skolnick; licensee BioMed Central Ltd.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2007</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Proteins</abbr></div>
                            <div id="Zhou200790" class="col-sm-8">
                                <div class="title">Analysis of TASSER-based CASP7 protein structure prediction
                                    results</div>
                                <div class="author"> H. Zhou, S.B. Pandit, Y.L. Seung, and <span class="more-authors"
                                        title="click to view 4 more authors"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'J. Borreguero, H. Chen, L. Wroblewska, J. Skolnick' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4
                                        more authors</span> </div>
                                <div class="periodical"> <em>Proteins: Structure, Function and Genetics</em> Oct
                                    2007 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1002/prot.21649" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>An improved TASSER (Threading/ASSEmbly/Refinement) methodology is applied to
                                        predict the tertiary structure for all CASP7 targets. TASSER employs
                                        template identification by threading, followed by tertiary structure
                                        assembly by rearranging continuous template fragments, where conformational
                                        space is searched via Parallel Hyperbolic Monte Carlo sampling with an
                                        optimized force-field that includes knowledge-based statistical potentials
                                        and restraints derived from threading templates. The final models are
                                        selected by clustering structures from the low temperature replicas.
                                        Improvements in TASSER over CASP6 involve use of better templates from
                                        3D-jury applied to three threading programs, PROSPECTOR_3, SP3, and SPARKS,
                                        and a fragment comparison method for better model ranking. For targets with
                                        no reliable templates, a variant of TASSER (chunk-TASSER) is also applied
                                        with potentials and restraints extracted from ab initio folded
                                        supersecondary chunks of the target to build full-length models. For all 124
                                        CASP targets/domains, the average root-mean-square-deviation (RMSD) from
                                        native and alignment coverage of the best initial threading models from
                                        3D-jury are 6.2 Å and 93%, respectively. Following TASSER reassembly, the
                                        average RMSD of the best model in the template aligned region decreases to
                                        4.9 Å and the average TM-score increases from 0.617 for the template to
                                        0.678 for the best full-length model. Based on target difficulty, the
                                        average TM-scores of the final model to native are 0.904, 0.671, and 0.307
                                        for high-accuracy template-based modeling, template-based modeling, and free
                                        modeling targets/domains, respectively. For the more difficult targets,
                                        TASSER with modest human intervention performed better in comparison to its
                                        server counterpart, MetaTASSER, which used a limited time simulation. © 2007
                                        Wiley-Liss, Inc.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Biochemistry</abbr></div>
                            <div id="Ding20072684" class="col-sm-8">
                                <div class="title">Phosphoprotein and phosphopeptide interactions with the FHA
                                    domain from arabidopsis kinase-associated protein phosphatase</div>
                                <div class="author"> Z. Ding, H. Wang, X. Liang, and <span class="more-authors"
                                        title="click to view 6 more authors"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'E.R. Morris, F. Gallazzi, S. Pandit, J. Skolnick, J.C. Walker, S.R. Van Doren' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6
                                        more authors</span> </div>
                                <div class="periodical"> <em>Biochemistry</em> Oct 2007 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1021/bi061763n" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>FHA domains are phosphoThr recognition modules found in diverse signaling
                                        proteins, including kinase-associated protein phosphatase (KAPP) from
                                        Arabidopsis thaliana. The kinase-interacting FHA domain (KI-FHA) of KAPP
                                        targets it to function as a negative regulator of some receptor-like kinase
                                        (RLK) signaling pathways important in plant development and environmental
                                        responses. To aid in the identification of potential binding sites for the
                                        KI-FHA domain, we predicted (i) the structure of a representative
                                        KAPP-binding RLK, CLAVATA1, and (ii) the functional surfaces of RLK kinase
                                        domains using evolutionary trace analysis. We selected phosphopeptides from
                                        KAPP-binding Arabidopsis RLKs for in vitro studies of association with
                                        KI-FHA from KAPP. Three phosphoThr peptide fragments from the kinase domain
                                        of CLV1 or BAK1 were found to bind KI-FHA with KD values of 8-20 μM, by NMR
                                        or titration calorimetry. Their affinity is driven by favorable enthalpy and
                                        solvation entropy gain. Mutagenesis of these three threonine sites suggests
                                        Thr546 in the C-lobe of the BAK1 kinase domain to be a principal but not
                                        sole site of KI-FHA binding in vitro. The brassinosteroid receptor BRI1 and
                                        KAPP are shown to associate in vivo and in vitro. Further genetic studies
                                        indicate that KAPP may be a negative regulator of the BRI1 signaling
                                        transduction pathway. l5N-Labeled KI-FHA was titrated with the GST-BRI1
                                        kinase domain and monitored by NMR. BRI1 interacts with the same 3/4, 4/5,
                                        6/7, 8/9, and 10/11 recognition loops of KI-FHA, with similar affinity as
                                        the phosphoThr peptides. © 2007 American Chemical Society.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2006</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Biophys. J.</abbr></div>
                            <div id="Pandit20064180" class="col-sm-8">
                                <div class="title">TASSER-lite: An automated tool for protein comparative modeling
                                </div>
                                <div class="author"> S.B. Pandit, Y. Zhang, and J. Skolnick</div>
                                <div class="periodical"> <em>Biophysical Journal</em> Oct 2006 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1529/biophysj.106.084293" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>This study involves the development of a rapid comparative modeling tool for
                                        homologous sequences by extension of the TASSER methodology, developed for
                                        tertiary structure prediction. This comparative modeling procedure was
                                        validated on a representative benchmark set of proteins in the Protein Data
                                        Bank composed of 901 single domain proteins (41-200 residues) having
                                        sequence identities between 35-90% with respect to the template. Using a
                                        Monte Carlo search scheme with the length of runs optimized for
                                        weakly/nonhomologous proteins, TASSER often provides appreciable improvement
                                        in structure quality over the initial template. However, on average, this
                                        requires ∼29 h of CPU time per sequence. Since homologous proteins are
                                        unlikely to require the extent of conformational search as
                                        weakly/nonhomologous proteins, TASSER’s parameters were optimized to reduce
                                        the required CPU time to ∼17 min, while retaining TASSER’s ability to
                                        improve structure quality. Using this optimized TASSER (TASSER-Lite), we
                                        find an average improvement in the aligned region of ∼10% in root
                                        mean-square deviation from native over the initial template. Comparison of
                                        TASSER-Lite with the widely used comparative modeling tool MODELLER showed
                                        that TASSER-Lite yields final models that are closer to the native.
                                        TASSER-Lite is provided on the web at
                                        http://cssb.biology.gatech.edu/skolnick/webservice/tasserlite/index. html. ©
                                        2006 by the Biophysical Society.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2005</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">In Silico Biol.</abbr></div>
                            <div id="Bhadra2005379" class="col-sm-8">
                                <div class="title">A new domain family in the superfamily of alkaline phosphatases
                                </div>
                                <div class="author"> R. Bhadra, N. Srinivasan, and S.B. Pandit</div>
                                <div class="periodical"> <em>In Silico Biology</em> Oct 2005 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>During the course of our large-scale genome analysis a conserved domain,
                                        currently detectable only in the genomes of Drosophila melanogaster,
                                        Caenorhabditis elegans and Anopheles gambiae, has been identified. The
                                        function of this domain is currently unknown and no function annotation is
                                        provided for this domain in the publicly available genomic, protein family
                                        and sequence databases. The search for the homologues of this domain in the
                                        non-redundant sequence database using PSI-BLAST, resulted in identification
                                        of distant relationship between this family and the alkaline
                                        phosphatase-like superfamily, which includes families of aryl sulfatase,
                                        N-acetylgalactosomine-4-sulfatase, alkaline phosphatase and
                                        2,3-bisphosphoglycerate-independent phosphoglycerate mutase (iPGM). The fold
                                        recognition procedures showed that this new domain could adopt a similar 3-D
                                        fold as for this supefamily. Most of the phosphatases and sulfatases of this
                                        superfamily are characterized by functional residues Ser and Cys
                                        respectively in the topologically equivalent positions. This functionally
                                        important site aligns with Ser/Thr in the members of the new family.
                                        Additionally, set of residues responsible for a metal binding site in
                                        phosphatases and sulphtases are conserved in the new family. The in-depth
                                        analysis suggests that the new family could possess phosphatase activity. ©
                                        2005 IOS Press. All rights reserved.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Nucleic Acids Res.</abbr></div>
                            <div id="Krishnadev2005" class="col-sm-8">
                                <div class="title">PRODOC: A resource for the comparison of tethered protein domain
                                    architectures with in-built information on remotely related domain families
                                </div>
                                <div class="author"> O. Krishnadev, N. Rekha, S.B. Pandit, and <span
                                        class="more-authors" title="click to view 5 more authors"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'S. Abhiman, S. Mohanty, L.S. Swapna, S. Gore, N. Srinivasan' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5
                                        more authors</span> </div>
                                <div class="periodical"> <em>Nucleic Acids Research</em> Oct 2005 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1093/nar/gki474" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>PROtein Domain Organization and Comparison (PRODOC) comprises several
                                        programs that enable convenient comparison of proteins as a sequence of
                                        domains. The in-built dataset currently consists of ∼698 000 proteins from
                                        192 organisms with complete genomic data, and all the SWISSPROT proteins
                                        obtained from the Pfam database. All the entries in PRODOC are represented
                                        as a sequence of functional domains, assigned using hidden Markov models,
                                        instead of as a sequence of amino acids. On average 69% of the proteins in
                                        the proteomes and 49% of the residues are covered by functional domain
                                        assignments. Software tools allow the user to query the dataset with a
                                        sequence of domains and identify proteins with the same or a jumbled or
                                        circularly permuted arrangement of domains. As it is proposed that proteins
                                        with jumbled or the same domain sequences have similar functions, this
                                        search tool is useful in assigning the overall function of a multi-domain
                                        protein. Unique features of PRODOC include the generation of alignments
                                        between multi-domain proteins on the basis of the sequence of domains and
                                        in-built information on distantly related domain families forming
                                        superfamilies. It is also possible using PRODOC to identify domain sharing
                                        and gene fusion events across organisms. An exhaustive genome-genome
                                        comparison tool in PRODOC also enables the detection of successive domain
                                        sharing and domain fusion events across two organisms. The tool permits the
                                        identification of gene clusters involved in similar biological processes in
                                        two closely related organisms. The URL for PRODOC is
                                        http://hodgkin.mbu.iisc.ernet.in/ prodoc. © 2005 Oxford University Press.
                                    </p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2004</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">In Silico Biol.</abbr></div>
                            <div id="Namboori2004445" class="col-sm-8">
                                <div class="title">Recognition of remotely related structural homologues using
                                    sequence profiles of aligned homologous protein structures</div>
                                <div class="author"> S. Namboori, N. Srinivasan, and S.B. Pandit</div>
                                <div class="periodical"> <em>In Silico Biology</em> Oct 2004 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>In order to bridge the gap between proteins with three-dimensional (3-D)
                                        structural information and those without 3-D structures, extensive
                                        experimental and computational efforts for structure recognition are being
                                        invested. One of the rapid and simple computational approaches for structure
                                        recognition makes use of sequence profiles with sensitive profile matching
                                        procedures to identify remotely related homologous families. While adopting
                                        this approach we used profiles that are generated from structure-based
                                        sequence alignment of homologous protein domains of known structures
                                        integrated with sequence homologues. We present an assessment of this fast
                                        and simple approach. About one year ago, using this approach, we had
                                        identified structural homologues for 315 sequence families, which were not
                                        known to have any 3-D structural information. The subsequent experimental
                                        structure determination for at least one of the members in 110 of 315
                                        sequence families allowed a retrospective assessment of the correctness of
                                        structure recognition. We demonstrate that correct folds are detected with
                                        an accuracy of 96.4% (106/110). Most (81/106) of the associations are made
                                        correctly to the specific structural family. For 23/106, the structure
                                        associations are valid at the superfamily level. Thus, profiles of protein
                                        families of known structure when used with sensitive profile-based search
                                        procedure result in structure association of high confidence. Further
                                        assignment at the level of superfamily or family would provide clues to
                                        probable functions of new proteins. Importantly, the public availability of
                                        these profiles from us could enable one to perform genome wide structure
                                        assignment in a local machine in a fast and accurate manner. © 2004 - IOS
                                        Press and Bioinformation Systems e.V. and the authors. All rights reserved.
                                    </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">In Silico Biol.</abbr></div>
                            <div id="Pandit2004563" class="col-sm-8">
                                <div class="title">Identification and analysis of a new family of bacterial serine
                                    proteinases</div>
                                <div class="author"> S.B. Pandit, and N. Srinivasan</div>
                                <div class="periodical"> <em>In Silico Biology</em> Oct 2004 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>A family of hypothetical proteins, identified predominantly from archaeal
                                        genomes, has been analyzed in order to understand its functional
                                        characteristics. Using extensive sequence similarity searches it is inferred
                                        that this family is remotely related (best sequence identity is 19%) to ClpP
                                        proteinases that belongs to serine proteinase class. This family of
                                        hypothetical proteins is referred to as SDH proteinase family based on
                                        conserved sequential order of Ser, Asp and His residues and predicted serine
                                        proteinase activity. Results of fold recognition of SDH family sequences
                                        confirmed the remote relationship between SDH proteinases and Clp
                                        proteinases and revealed similar tertiary location of putative catalytic
                                        triad residues critical for serine proteinase function. However, the best
                                        sequence alignment we could obtain suggests that while catalytic Ser is
                                        conserved across Clp and SDH proteinases the location of the other catalytic
                                        triad residues, namely, His and Asp are swapped in their amino acid
                                        alignment positions and hence in 3-D structure. The evidence of conserved
                                        catalytic triad suggests that SDH could be a new family of serine
                                        proteinases with the fold of Clp proteinase, however sharing the catalytic
                                        triad order of carboxypeptidase clan. Signal peptide sequence identified at
                                        the N-terminus of some of the homologues suggests that these might be
                                        secretory serine proteinases involved in cleavage of extracellular proteins
                                        while the remote homologues, ClpP proteinases, are known to work in
                                        intracellular environment. © 2004 - IOS Press and Bioinformation System e.V.
                                        and the authors. All rights reserved.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">IUBMB Life</abbr></div>
                            <div id="Pandit2004317" class="col-sm-8">
                                <div class="title">Structural and functional characterization of gene products
                                    encoded in the human genome by homology detection</div>
                                <div class="author"> S.B. Pandit, S. Balaji, and N. Srinivasan</div>
                                <div class="periodical"> <em>IUBMB Life</em> Oct 2004 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1080/15216540400006105" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Availability of the human genome data has enabled the exploration of a huge
                                        amount of biological information encoded in it. There are extensive ongoing
                                        experimental efforts to understand the biological functions of the gene
                                        products encoded in the human genome. However, computational analysis can
                                        aid immensely in the interpretation of biological function by associating
                                        known functional/structural domains to the human proteins. In this article
                                        we have discussed the implications of such associations. The association of
                                        structural domains to human proteins could help in prioritizing the targets
                                        for structure determination in the structural genomics initiatives. The
                                        protein kinase family is one of the most frequently occurring protein domain
                                        families in the human proteome while P-loop hydrolase, which comprises many
                                        GTPases and ATPases, is a highly represented superfamily. Using the
                                        superfamily relationships between families of unknown and known structures
                                        we could increase structural information content of the human genome by
                                        about 5%. We could also make new associations of domain families to 33 human
                                        proteins that are potentially linked to genetically inherited diseases.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">BMC Bioinform.</abbr></div>
                            <div id="Pandit2004" class="col-sm-8">
                                <div class="title">SUPFAM: A database of sequence superfamilies of protein domains
                                </div>
                                <div class="author"> S.B. Pandit, R. Bhadra, V.S. Gowri, and <span class="more-authors"
                                        title="click to view 3 more authors"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'S. Balaji, B. Anand, N. Srinivasan' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3
                                        more authors</span> </div>
                                <div class="periodical"> <em>BMC Bioinformatics</em> Oct 2004 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1186/1471-2105-5-28" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Background: SUPFAM database is a compilation of superfamily relationships
                                        between protein domain families of either known or unknown 3-D structure. In
                                        SUPFAM, sequence families from Pfam and structural families from SCOP are
                                        associated, using profile matching, to result in sequence superfamilies of
                                        known structure. Subsequently all-against-all family profile matches are
                                        made to deduce a list of new potential superfamilies of yet unknown
                                        structure. Description: The current version of SUPFAM (release 1.4)
                                        corresponds to significant enhancements and major developments compared to
                                        the earlier and basic version. In the present version we have used
                                        RPS-BLAST, which is robust and sensitive, for profile matching. The
                                        reliability of connections between protein families is ensured better than
                                        before by use of benchmarked criteria involving strict e-value cut-off and a
                                        minimal alignment length condition. An e-value based indication of
                                        reliability of connections is now presented in the database. Web access to a
                                        RPSBLAST-based tool to associate a query sequence to one of the family
                                        profiles in SUPFAM is available with the current release. In terms of the
                                        scientific content the present release of SUPFAM is entirely reorganized
                                        with the use of 6190 Pfam families and 2317 structural families derived from
                                        SCOP. Due to a steep increase in the number of sequence and structural
                                        families used in SUPFAM the details of scientific content in the present
                                        release are almost entirely complementary to previous basic version. Of the
                                        2286 families, we could relate 245 Pfam families with apparently no
                                        structural information to families of known 3-D structures, thus resulting
                                        in the identification of new families in the existing superfamilies. Using
                                        the profiles of 3904 Pfam families of yet unknown structure, an
                                        all-against-all comparison involving sequence-profile match resulted in
                                        clustering of 96 Pfam families into 39 new potential superfamilies.
                                        Conclusion: SUPFAM presents many non-trivial superfamily relationships of
                                        sequence families involved in a variety of functions and hence the
                                        information content is of interest to a wide scientific community. The
                                        grouping of related proteins without a known structure in SUPFAM is useful
                                        in identifying priority targets for structural genomics initiatives and in
                                        the assignment of putative functions. © 2004 Pandit et al; licensee BioMed
                                        Central Ltd.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">J. Biosci.</abbr></div>
                            <div id="Namboori2004245" class="col-sm-8">
                                <div class="title">Enhanced functional and structural domain assignments using
                                    remote similarity detection procedures for proteins encoded in the genome of
                                    Mycobacterium tuberculosis H37Rv</div>
                                <div class="author"> S. Namboori, N. Mhatre, S. Sujatha, and <span class="more-authors"
                                        title="click to view 2 more authors"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'N. Srinivasan, S.B. Pandit' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2
                                        more authors</span> </div>
                                <div class="periodical"> <em>Journal of Biosciences</em> Oct 2004 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1007/BF02702607" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>The sequencing of the Mycobacterium tuberculosis (MTB) H37Rv genome has
                                        facilitated deeper insights into the biology of MTB, yet the functions of
                                        many MTB proteins are unknown. We have used sensitive profile-based search
                                        procedures to assign functional and structural domains to infer functions of
                                        gene products encoded in MTB. These domain assignments have been made using
                                        a compendium of sequence and structural domain families. Functions are
                                        predicted for 78% of the encoded gene products. For 69% of these, functions
                                        can be inferred by domain assignments. The functions for the rest are
                                        deduced from their homology to proteins of known function. Superfamily
                                        relationships between families of unknown and known structures have
                                        increased structural information by ∼ 11%. Remote similarity detection
                                        methods have enabled domain assignments for 1325 ’hypothetical proteins’.
                                        The most populated families in MTB are involved in lipid metabolism, entry
                                        and survival of the bacillus in host. Interestingly, for 353 proteins,
                                        which, we refer to as MTB-specific, no homologues have been identified.
                                        Numerous, previously unannotated, hypothetical proteins have been assigned
                                        domains and some of these could perhaps be the possible chemotherapeutic
                                        targets. MTB-specific proteins might include factors responsible for
                                        virulence. Importantly, these assignments could be valuable for experimental
                                        endeavors. The detailed results are publicly available at
                                        http://hodgkin.mbu.iisc.ernet.in/ dots.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2003</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Proteins</abbr></div>
                            <div id="Pandit2003585" class="col-sm-8">
                                <div class="title">Survey for G-proteins in the prokaryotic genomes: Prediction of
                                    functional roles based on classification</div>
                                <div class="author"> S.B. Pandit, and N. Srinivasan</div>
                                <div class="periodical"> <em>Proteins: Structure, Function and Genetics</em> Oct
                                    2003 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1002/prot.10420" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>The members of the family of G-proteins are characterized by their ability to
                                        bind and hydrolyze guanosine triphosphate (GTP) to guanosine diphosphate
                                        (GDP). Despite a common biochemical function of GTP hydrolysis shared among
                                        the members of the family of G-proteins, they are associated with diverse
                                        biological roles. The current work describes the identification and detailed
                                        analysis of the putative G-proteins encoded in the completely sequenced
                                        prokaryotic genomes. Inferences on the biological roles of these G-proteins
                                        have been obtained by their classification into known functional
                                        subfamilies. We have identified 497 G-proteins in 42 genomes. Seven small
                                        GTP-binding protein homologues have been identified in prokaryotes with at
                                        least two of the diagnostic sequence motifs of G-proteins conserved. The
                                        translation factors have the largest representation (234 sequences) and are
                                        found to be ubiquitous, which is consistent with their critical role in
                                        protein synthesis. The GTP_OBG subfamily comprises of 79 sequences in our
                                        dataset. A total of 177 sequences belong to the subfamily of GTPase of
                                        unknown function and 154 of these could be associated with domains of known
                                        functions such as cell cycle regulation and t-RNA modification. The large
                                        GTP-binding proteins and the α-subunit of heterotrimeric G-proteins are not
                                        detected in the genomes of the prokaryotes surveyed. © 2003 Wiley-Liss, Inc.
                                    </p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Nucleic Acids Res.</abbr></div>
                            <div id="Gowri2003486" class="col-sm-8">
                                <div class="title">Integration of related sequences with protein three-dimensional
                                    structural families in an updated version of PALI database</div>
                                <div class="author"> V.S. Gowri, S.B. Pandit, P.S. Karthik, and <span
                                        class="more-authors" title="click to view 2 more authors"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'N. Srinivasan, S. Balaji' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2
                                        more authors</span> </div>
                                <div class="periodical"> <em>Nucleic Acids Research</em> Oct 2003 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1093/nar/gkg063" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>The database of Phylogeny and ALIgnment of homologous protein structures
                                        (PALI) contains three-dimensional (3-D) structure-dependent sequence
                                        alignments as well as structure-based phylogenetic trees of protein domains
                                        in various families. The latest updated version (Release 2.1) comprises of
                                        844 families of homologous proteins involving 3863 protein domain structures
                                        with each of these families having at least two members. Each member in a
                                        family has been structurally aligned with every other member in the same
                                        family using two proteins at a time. In addition, an alignment of multiple
                                        structures has also been performed using all the members in a family. Every
                                        family with at least three members is associated with two dendrograms, one
                                        based on a structural dissimilarity metric and the other based on similarity
                                        of topologically equivalenced residues for every pairwise alignment. Apart
                                        from these multi-member families, there are 817 single member families in
                                        the updated version of PALI. A new feature in the current release of PALI is
                                        the integration, with 3-D structural families, of sequences of homologues
                                        from the sequence databases. Alignments between homologous proteins of known
                                        3-D structure and those without an experimentally derived structure are also
                                        provided for every family in the enhanced version of PALI. The database with
                                        several web interfaced utilities can be accessed at:
                                        http://pauling.mbu.iisc.ernet.in/ pali.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
                <h2 class="year">2002</h2>
                <ol class="bibliography">
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Nucleic Acids Res.</abbr></div>
                            <div id="Pandit2002289" class="col-sm-8">
                                <div class="title">SUPFAM - A database of potential protein superfamily
                                    relationships derived by comparing sequence-based and structure-based families:
                                    Implications for structural genomics and function annotation in genomes</div>
                                <div class="author"> S.B. Pandit, D. Gosar, S. Abhiman, and <span class="more-authors"
                                        title="click to view 5 more authors"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'S. Sujatha, S.S. Dixit, N.S. Mhatre, R. Sowdhamini, N. Srinivasan' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5
                                        more authors</span> </div>
                                <div class="periodical"> <em>Nucleic Acids Research</em> Oct 2002 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1093/nar/30.1.289" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Members of a superfamily of proteins could result from divergent evolution of
                                        homologues with insignificant similarity in the amino acid sequences. A
                                        superfamily relationship is detected commonly after the three-dimensional
                                        structures of the proteins are determined using X-ray analysis or NMR. The
                                        SUPFAM database described here relates two homologous protein families in a
                                        multiple sequence alignment database of either known or unknown structure.
                                        The present release (1.1), which is the first version of the SUPFAM
                                        database, has been derived by analysing Pfam, which is one of the commonly
                                        used databases of multiple sequence alignments of homologous proteins. The
                                        first step in establishing SUPFAM is to relate Pfam families with the
                                        families in PALI, which is an alignment database of homologous proteins of
                                        known structure that is derived largely from SCOP. The second step involves
                                        relating Pfam families which could not be associated reliably with a protein
                                        superfamily of known structure. The profile matching procedure, IMPALA, has
                                        been used in these steps. The first step resulted in identification of 1280
                                        Pfam families (out of 2697, i.e. 47%) which are related, either by close
                                        homologous connection to a SCOP family or by distant relationship to a SCOP
                                        family, potentially forming new superfamily connections. Using the profiles
                                        of 1417 Pfam families with apparently no structural information, an
                                        all-against-all comparison involving a sequence-profile match using IMPALA
                                        resulted in clustering of 67 homologous protein families of Pfam into 28
                                        potential new superfamilies. Expansion of groups of related proteins of yet
                                        unknown structural information, as proposed in SUPFAM, should help in
                                        identifying ’priority proteins’ for structure determination in structural
                                        genomics initiatives to expand the coverage of structural information in the
                                        protein sequence space. For example, we could assign 858 distinct Pfam
                                        domains in 2203 of the gene products in the genome of Mycobacterium
                                        tubercolosis. Fifty-one of these Pfam families of unknown structure could be
                                        clustered into 17 potentially new superfamilies forming good targets for
                                        structural genomics. SUPFAM database can be accessed at
                                        http://pauling.mbu.iisc.ernet.in/ supfam.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                    <li>
                        <div class="row">
                            <div class="col-sm-2 abbr"><abbr class="badge">Protein Eng.</abbr></div>
                            <div id="Gupta2002503" class="col-sm-8">
                                <div class="title">Proteomics analysis of carbon-starved Mycobacterium smegmatis:
                                    Induction of Dps-like protein</div>
                                <div class="author"> S. Gupta, S.B. Pandit, N. Srinivasan, and <span
                                        class="more-authors" title="click to view 1 more author"
                                        onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'D. Chatterji' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">1
                                        more author</span> </div>
                                <div class="periodical"> <em>Protein Engineering</em> Oct 2002 </div>
                                <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
                                    <a href="http://doi.org/10.1093/protein/15.6.503" class="btn btn-sm z-depth-0"
                                        role="button" target="_blank" rel="noopener noreferrer">HTML</a>
                                </div>
                                <div class="abstract hidden">
                                    <p>Mycobacterium tuberculosis is a globally successful pathogen, infecting more
                                        than one third of total world’s population. These bacteria have the
                                        remarkable ability to persist in the host for long periods of time
                                        unrecognized by the immune system and then to re-emerge later in life
                                        causing the disease. The physiology of such persistent or dormant bacilli is
                                        not very well characterized. Some evidence suggests that the dormant bacilli
                                        survive in a nutrient-deprived state that is similar to the stationary phase
                                        of the bacteria with respect to gene expression and physiology. Under this
                                        assumption we have studied the survival of Mycobacterium smegmatis in carbon
                                        starvation conditions as a model for mycobacterial persistence. M.smegmatis,
                                        being a fast-growing strain, serves as a good model to study starvation
                                        responses. Using the two-dimensional electrophoresis-based proteomics
                                        approach, we identified a protein which was found to be expressed
                                        preferentially under starvation conditions. This protein is homologous to a
                                        family of proteins called Dps (DNA binding Protein from Starved cells) that
                                        are known to protect DNA under various kinds of environmental stresses and
                                        its existence has, so far, not been reported in mycobacteria. Upon
                                        expression and purification of this protein, we observed that it has
                                        non-specific DNA-binding ability. Formation of a cage-like dodecamer
                                        structure is a characteristic feature of Dps. Using comparative modelling we
                                        were able to show that Dps from M.smegmatis could form a dodecamer structure
                                        similar to the crystal structure of Dps from Escherichia coli.</p>
                                </div>
                            </div>
                        </div>
                    </li>
                </ol>
        </div>
        </article>
    </div>
    </div>
    <footer class="fixed-bottom">
        <div class="container mt-0"> © Copyright 2024 Shashi B. Pandit. Powered by <a href="https://jekyllrb.com/"
                target="_blank" rel="noopener noreferrer">Jekyll</a> with <a
                href="https://github.com/alshedivat/al-folio" target="_blank" rel="noopener noreferrer">al-folio</a>
            theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="noopener noreferrer">GitHub
                Pages</a>. </div>
    </footer>
    <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js"
        integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js"
        integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script>
    <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js"
        integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>
    <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js"
        integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
    <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
    <script defer src="/assets/js/masonry.js" type="text/javascript"></script>
    <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js"
        integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
    <script defer src="/assets/js/zoom.js"></script>
    <script defer src="/assets/js/common.js"></script>
    <script type="text/javascript">window.MathJax = { tex: { tags: "ams" } };</script>
    <script defer type="text/javascript" id="MathJax-script"
        src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
    <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
</body>

</html>
